_episodes/02-variables.md

```diff
@@ -153,7 +153,7 @@ h
 *   We take a slice by using `[start:stop]`, where `start` is replaced with the
     index of the first element we want and `stop` is replaced with the index of
     the element just after the last element we want.
-*   Mathematically, you might say that a slice selects `[start:stop)`.
+*   Mathematically, you might say that a slice selects `[start:stop]`.
```

Comment:
The `[start:stop)` expression (which is invalid Python syntax) refers to [mathematical interval notation](https://en.wikipedia.org/wiki/Interval_(mathematics)#Notations_for_intervals) and means "include the value at `start`, but do not include the value at `stop`". If this is confusing to others however I'm open to adding a footnote or link or finding a different way to explain it.